src/plugins/aws/ssh.ts

```diff
@@ -113,6 +114,12 @@ export const awsSshProvider: SshProvider<
     return undefined;
   },
 
+  generateKeys: async (_) => {
+    return {
+      privateKeyPath: path.join(P0_PATH, "ssh", "id_rsa"),
```

Comment:
DRY should we be using: `PRIVATE_KEY_PATH` here instead?  or should we be calling this method anywhere where we need to access the private key path for an ssh provider?